.env.template

```
@@ -0,0 +1,20 @@
+REST_PORT=3000
+RULE_VERSION=1.0
+LOGSTASH_HOST=my-release-logstash.frm-meshed
+LOGSTASH_PORT=8080
+APM_LOGGING=true
+FUNCTION_NAME=rule-028
+APM_URL=
+APM_SECRET_TOKEN=
+NODE_ENV=dev
+DATABASE_NAME=Configuration
+DATABASE_URL=
+DATABASE_USER=root
+DATABASE_PASSWORD=$!prAtHe>Qh5X9D3
```

Please do not share tokens / passwords in ANYTHING we commit. 

```suggestion
DATABASE_PASSWORD=
```